src/styles/variables.scss

```diff
@@ -59,7 +59,7 @@ body {
   // sizes
   --container-width: 1200px;
   --container-padding: 32px;
-  --header-height: 61px;
+  --header-height: 77px;
```

Comment:
This variable is only used with your records list, but not with the header itself. So currently header's height is differ from that value

Reply:
And what i need to do with that variable, like, create new for this record list?